tool/test-annocheck.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh -eu
+# Run the `tool/test-annocheck.sh [binary files]` to check security issues.
+# E.g. `tool/test-annocheck.sh ruby libruby.so.3.2.0`.
+#
+# Use annocheck.
+# https://sourceware.org/annobin/
+
+set -x
+
+DOCKER="$(command -v docker || command -v podman)"
+TAG=ruby-fedora-annocheck
+TOOL_DIR=$(dirname "${0}")
+
+"${DOCKER}" build --rm -t "${TAG}" ${TOOL_DIR}/annocheck/
+"${DOCKER}" run --rm -t -v $(pwd):/work "${TAG}" annocheck --verbose ${@}
```

Comment:
`${@}` without double-qoutes equals `${*}`, `"${@}"` is different from `"${*}"`.
```suggestion
"${DOCKER}" run --rm -t -v $(pwd):/work "${TAG}" annocheck --verbose "${@}"
```

Reply:
ok, thanks! I applied the change.